.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] As stated the project requirement kindly make sure that you added A `test.md` file with the text "This file should be ignored by git" and  is actually ignored by git and is not present in your GitHub repository.

![Capture](https://user-images.githubusercontent.com/93598189/204548317-cd6b906c-fac9-4daf-8dc0-698406380c12.PNG)